src/main/kotlin/net/fabricmc/bot/extensions/mappings/MappingsManager.kt

```diff
@@ -76,11 +78,34 @@ class MappingsManager {
         val mappings = openMappings(minecraftVersion) ?: return null
 
         return mappings.classes
-                .filter { matches(it, query) }
+                .filter { matches(it, preProcessClassQuery(query)) }
```

Comment:
Super pedantic, but this does mean that we're calling `preProcessClassQuery` for each filter iteration — might be best to store the pre-processed query in a variable before the return